ecc/Makefile

```diff
@@ -56,6 +58,10 @@ clean: docker-clean
 #	rm enclave/mrenclave.go
 
 docker:
+	@if [ -z "${DOCKER_IMAGE}" ]; then\
+		echo "\033[0;31mERROR: no $(CC_NAME) docker image found\033[0m";\
```

Comment:
This error message seems to be not super intuitive. As `make docker` creates a docker image for CC_NAME, the error tells the user that here is no such image. mhhh.

I suggest the following.

If there is no ercc docker image yet, output a warning (maybe in yellow if you want) and create the image with some default name which can be set by the user.

If the docker image exists, maybe also output a warning that we override an existing image.

WDYT?

Reply:
I changed the message.
For the rest I am not sure what the "current" use of that would be. I would postpone it to a separate PR that addresses this.

Reply:
ok for me